cmd/snap-bootstrap/cmd_create_partitions.go

```diff
@@ -20,8 +20,12 @@
 package main
 
 import (
+	"fmt"
```

Comment:
This entire file moves into a test only helper in PR#8551 - if we spread test this new feature I will make sure that this code is moved in 8551 too, otherwise (if we don't want to spread test it) we can remove those parts in here now

Reply:
Samuele answered this, want this and a spread test that measures the model

Reply:
I see the move in #8551, I think it would be nice to have the model in the spread test too.